src/openrct2/interface/console.c

```diff
@@ -399,8 +413,7 @@ void console_scroll(sint32 linesToScroll)
 // Calculates the amount of visible lines, based on the console size, excluding the input line.
 static sint32 console_get_num_visible_lines()
 {
-    const sint32 lineHeight = 10;
-    return ((_consoleBottom - 22 - _consoleTop) / lineHeight) - 1;
+    return ceil((_consoleBottom - _consoleTop) / font_get_line_height(gCurrentFontSpriteBase)) - 1;
```

Comment:
All available `ceil` signatures take some real number and return a real number. We neither pass in a real value, nor expect one to be returned. Please fix that to use integer maths.